tools/test/audio/std_utils/thdnf_test_measure.m

```diff
@@ -72,16 +72,16 @@
 	test.thdnf_high(:,i) = test.thdnf(:,1);
 	test.thdnf_low(:,i) = test.thdnf(:,2);
 
-	fidx = find(test.thdnf(idx, 1) > mask_hi);
-	if length(fidx) > 0
+	fidx = find(test.thdnf(idx, 1) > mask_hi, 1);
+	if ~isempty(fidx)
```

Comment:
Is this for Matlab compatibility? Looks more like cleanup, though I'm not 100% sure. Please check.

Reply:
`isempty` is faster than comparing `length` to `0`. Since it is a general performance improvement, compatibility section does not apply to it.